packages/uiweb/src/lib/components/space/SpacesUI.tsx

```diff
@@ -4,6 +4,7 @@ import { ISpaceBannerProps, SpaceBanner } from './SpaceBanner';
 import { ISpaceWidgetProps, SpaceWidget } from './SpaceWidget';
```

Comment:
not able to test the PR yet, due to circular dependency with demoreact: Circular dependency between "demoreact" and "uiweb" detected: demoreact -> uiweb -> demoreact

Circular file chain:
- packages/demoreact/src/app/SpaceUITest/useSpaceComponents.tsx
- packages/uiweb/src/lib/components/space/SpaceCreationWidget/SpaceCreationWidget.tsxeslint[@nrwl/nx/enforce-module-boundaries](file:///)

also, need to add testing page for this component in demoreact